lib/blacklight/canvas_course.rb

```diff
@@ -54,7 +65,72 @@ def self.from_metadata(metadata)
             name: course_name,
           },
         )
-      CanvasCourse.new(metadata, canvas_course)
+      CanvasCourse.new(metadata, canvas_course, blackboard_export)
+    end
+
+    ##
+    # Creates a canvas assignment from a scorm package that has already been
+    # uploaded to a scorm manager
+    ##
+    def create_scorm_assignment(scorm_package, course_id)
+      url = "#{Blacklight.scorm_launch_url}?" +
```

Comment:
Consider putting the question mark in the second string, and referencing `Blacklight.scorm_launch_url` directly, instead of via string interpolation, like this:

```ruby
url = Blacklight.scorm_launch_url +
        "?course_id=#{scorm_package['package_id']}"
```
Maybe that would even make the line short enough that it doesn't need to wrap.